installInstructions/install3rdPartyPackages.rst

```diff
@@ -428,94 +371,85 @@ Python Packages
 Other methods, such as from source or using ``conda``, will also work.
 Local installations (with ``--user``) are also recommended but not required.
-If pip is not available, install it using:
-
-.. code-block:: bash
-
-   cd $HOME/PACKAGES
-   wget https://bootstrap.pypa.io/get-pip.py
-   python get-pip.py --user
 
 When installing the same package multiple times with different dependencies,
 for example ``petsc4py`` with different petsc builds, the pip cache can become incorrect.
 Therefore, we recommend the ``--no-cache`` flag when installing python packages with pip.
 
-.. _install_num_sci_py:
-
 .. _install_numpy:
 
-`Numpy <https://numpy.org/>`__
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+NumPy
+~~~~~
 
 .. IMPORTANT::
    Version ``1.13.3`` and ``1.15.4`` of numpy or f2py do **NOT** work.
-   See :ref:`working_stacks` for numpy versions that have been tested.
-   The version(s) of numpy used here is ``1.16.4``.
+   See :ref:`working_stacks` for numpy versions that are tested.
 
-Numpy is required for all MDO Lab packages.
+NumPy is required for all MDO Lab packages.
 It is installed with::
 
-   pip install numpy==1.16.4 --user --no-cache
+   pip install numpy==<version> --user --no-cache
 
 On a ``conda``-based system, it is recommended to use ``conda`` to install numpy and scipy::
 
-   conda install numpy==1.16.4
+   conda install numpy==<version>
 
-`Scipy <http://scipy.org/>`__
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Scipy is required for several packages including ``pyOptSparse``, ``pyGeo`` and certain functionality in ``pySpline``.
+SciPy
+~~~~~
+SciPy is required for several packages including ``pyOptSparse``, ``pyGeo`` and certain functionality in ``pySpline``.
 It is installed with::
 
-   pip install scipy==1.2.1 --user --no-cache
+   pip install scipy==<version> --user --no-cache
 
 On a ``conda``-based system, it is recommended to use ``conda`` to install numpy and scipy::
 
-   conda install scipy==1.2.1
+   conda install scipy==<version>
 
 .. note::
    On a cluster, most likely numpy and scipy will already be
    installed. Unless the version is invalid, use the system-provided installation.
 
 .. _install_mpi4py:
 
-`mpi4py <http://mpi4py.scipy.org/>`__
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+mpi4py
+~~~~~~
 .. IMPORTANT::
-   The version(s) of mpi4py tested to work with MDO Lab tools is 3.0.2.
-
    mpi4py depends on OpenMPI.
-
    It is recommended that the OpenMPI version matches with the mpi4py version.
 
 mpi4py is the Python wrapper for MPI. This is required for
 **all** parallel MDO Lab codes.
 It is installed with::
 
-   pip install mpi4py==3.0.2 --user --no-cache
+   pip install mpi4py==<version> --user --no-cache
 
 .. NOTE::
    Some function usages have changed in newer versions of mpi4py. Check the `release <https://github.com/mpi4py/mpi4py/blob/master/CHANGES.rst>`_ to see the modifications that might be requried in the code.
 
+Alternatively, downloading from source is also possible.
```

Comment:
I think I would separate this information under its own heading like `Alternative installation` or `Advanced installation` to separate what most people want (which is pip) from what a more specialized install would look like.
This also aligns then nicely with `petsc4py` structure.